client/src/components/AboutSection.css

```diff
@@ -1,33 +1,38 @@
-.about-section {
-	height: 544px;
+.about-background {
 	background-color: #000;
-	display: flex;
-	flex-direction: row;
-	justify-content: center;
+	width: 100%;
+}
+
+.about-section {
+	margin: auto;
+	padding: 0 16px;
```

Comment:
Maybe swap the margin-top and margin-bottom for a (as discussed) 60px padding top/bottom on mobile, 16-20px padding top/bottom on desktop